main.py

```diff
@@ -14,9 +14,11 @@
 from matplotlib import pyplot as plt
 from sklearn.tree import DecisionTreeRegressor
 from sklearn.svm import SVC
+from sklearn.linear_model import LinearRegression
```

Comment:
We won't send this file next time.. should we update it?

Reply:
keep it for debugging and testing until the final review